FAQ/en.md

```diff
@@ -0,0 +1,73 @@
+# Frequently Asked Questions
+
+## General
+
+### What is Quaver?
+
+Quaver is a community-driven rhythm game, aiming to bring VSGRs to the next level. The game is open to players of all skill levels, casual and seasoned.
+
+### How is Quaver different from other games?
+
+The game is fully open-source, allowing anyone to help and improve the game. Furthermore, we are aiming to provide a new VSGR experience by bringing competitive concepts, such as competitive matchmaking.
+
+### How far in development is Quaver?
+
+Quaver is still in early development and is currently in closed alpha. In order to use online features of Quaver, you need to sign up for the alpha.
+
+### What platforms will Quaver be released on?
+
+Quaver will be released on Windows, Mac and Linux.
+
+### Where can I sign up for the Quaver Alpha?
+
+Registrations for the closed alpha occur in rounds. Each round of sign-ups are announced before they open.
+
+### Can I play without being part of the Quaver Alpha?
+
+Yes, you can download an offline version from Quaver’s [Github page](https://github.com/Quaver/Quaver/releases), However, you won’t be able to experience any of the online features including online rankings, map uploading, and other community-based features such as in-game chat.
+
+### Does this mean I can’t download maps from the Quaver website?
+
+You cannot download maps from the Quaver website, however you can still download packs via a [mirror](https://rhythmgamers.net/pack/).
+
+### Is there an estimate for the public release date?
+
+We do not have an estimate for the public release date yet.
+
+
```

Comment:
Remove the extra line break here

Reply:
Also removed the line break above Troubleshooting.